tools/lsp/preview.rs

```diff
@@ -250,7 +250,6 @@ fn search_for_parent_element(root: &ElementRc, child: &ElementRc) -> Option<Elem
     None
 }
 
-// triggered from the UI, running in UI thread
```

Comment:
I suggest adding a module documentation that says all function must be called from the UI thread

Reply:
Added.